<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <arg name="robot" default="/"/>
    <arg name="simulate" default="False"/>
	<group ns="$(arg robot)">

		<!-- Color detection node -->
		<node name="behavior" pkg="motivational_dms" type="behavior_manager.py" output="screen">
		</node>
		<!-- Detect user position node -->
		<node name="decision_making" pkg="motivational_dms" type="decision_making_system.py" output="screen">
			<param name="simulate" value="$(arg simulate)" /> 
		</node>

	</group>

</launch>